1.10/installing/custom/configuration/configuration-parameters.md

```diff
@@ -41,6 +41,7 @@ This topic provides all available configuration parameters. Except where explici
 | [dns_forward_zones](#dns_forward_zones)               | A nested list of DNS zones, IP addresses, and ports that configure custom forwarding behavior of DNS queries. A DNS zone is mapped to a set of DNS resolvers. |
 | [dns_search](#dns_search)                             | A space-separated list of domains that are tried when an unqualified domain is entered.  |
 | [master_dns_bindall](#master_dns_bindall)             | Indicates whether the master DNS port is open.  |
+| [mesos_dns_set_truncate_bit](#mesos_dns_set_truncate_bit)   |  Indicates whether to set the truncate bit if the message is too large to fit in the response. |
```

Comment:
s/if the message is too large to fit in the response./if the response is too large to fit in a single packet. /